resources/multisite_drupal_standard.make

```diff
@@ -789,15 +789,7 @@ projects[workbench_og][version] = "2.0-beta1"
 projects[workbench_og][patch][] = https://www.drupal.org/files/issues/workbench_og-node_access-2835937.patch
 
 projects[wysiwyg][subdir] = "contrib"
-projects[wysiwyg][version] = "2.2"
-projects[wysiwyg][patch][] = patches/wysiwyg-ckeditor4-bug-version-1799.patch
-projects[wysiwyg][patch][] = patches/wysiwyg-ckeditor_ie_fix-1914904-5.patch
-projects[wysiwyg][patch][] = patches/wysiwyg-local_css_file_paths-1793704-14.patch
-projects[wysiwyg][patch][] = patches/wysiwyg-js-url-9874.patch
```

Comment:
Why we remove this patch? it's not included in the version 2.4.

Reply:
My understanding is that it's no longer required after this patch was committed in 
https://github.com/ec-europa/platform-dev/pull/736/files

Reply:
Maybe, we need double check this affirmation.

Reply:
@voidtek yes i asked @jfhovinne  to do this, he does not have time, i hope someone else will.